runtime/src/snapshot_utils/archive_format.rs

```diff
@@ -89,6 +93,13 @@ impl fmt::Display for ParseError {
     }
 }
 
+/// Configuration when using zstd as the snapshot archive format
+#[derive(Debug, Default, Copy, Clone, Eq, PartialEq)]
```

Comment:
`i32::default()` is `0`, but we use `1` today, no? IMO, a `Default` impl really doesn't make sense on this type at all. we only want the concept of "default" in the context of the cli config

Reply:
if it's not going to cause conflicts, i guess it can ride master